landing/templates/landing/learn-more.html

```diff
@@ -42,41 +42,41 @@ <h2>Pro4U <br><span>Innovative Solution!</span></h2>
 		<div class="team-members">
```

Comment:
Does it make any sense to use a conventional [for loop](https://docs.djangoproject.com/en/4.2/ref/templates/builtins/#for) here? Tomorrow, if someone leaves/joins the team, it can be added/removed dynamically

Reply:
> Does it make any sense to use a conventional [for loop](https://docs.djangoproject.com/en/4.2/ref/templates/builtins/#for) here? Tomorrow, if someone leaves/joins the team, it can be added/removed dynamically

Resolved